include/capstone/sparc.h

```diff
@@ -204,6 +176,34 @@ typedef enum sparc_reg {
 	SPARC_REG_I6 = SPARC_REG_FP,
 } sparc_reg;
 
+// Instruction's operand referring to memory
+// This is associated with SPARC_OP_MEM operand type above
+typedef struct sparc_op_mem {
+	sparc_reg base;		// base register
```

Comment:
this change will break all the current bindings because it changes the structure size from uint8_t to int